README.md

```diff
@@ -1,7 +1,10 @@
 jBBCode
 =======
+[![GitHub release](https://img.shields.io/github/release/jbowens/jBBCode.svg)](https://github.com/jbowens/jBBCode/releases)
```

Comment:
Hm yes, you're probably right. I openly admit, adding the VersionEye badge has been  more of a "because I can!" But it really doesn't add much value. STill find the number of installs neat, but if you don't like it ... I'll remove the two right away :)